roles/ovirt-hosts/README.md

```diff
@@ -14,15 +14,16 @@ Role Variables
 
 The `hosts` list can contain the following parameters:
 
-| Name          | Default value  | Description                           |
-|---------------|----------------|---------------------------------------|
-| name          | UNDEF          | Name of the host.                      |
-| state         | present        | Specifies whether the host is `present` or `absent`.  |
-| address       | UNDEF          | IP address or FQDN of the host.   |
-| password      | UNDEF          | The host's root password.             |
-| cluster       | UNDEF          | The cluster that the host must connect to.    |
-| timeout       | 1200           | Maximum wait time for the host to be in an UP state.  |
-| poll_interval | 20             | Polling interval to check the host status. |
+| Name          | Default value    | Description                           |
+|---------------|------------------|---------------------------------------|
+| name          | UNDEF (Required) | Name of the host.                      |
+| state         | present          | Specifies whether the host is `present` or `absent`.  |
+| address       | UNDEF (Required) | IP address or FQDN of the host.   |
+| password      | UNDEF            | The host's root password. Required if <i>public_key</i> is false. |
+| public_key    | UNDEF            | If <i>true</i> the public key should be used to authenticate to host. |
```

Comment:
I thought that this param will contain the public key of the host. So why boolean?

Reply:
No, it's true/false, we don't support specify custom public key, only the default one - the one deployed by host-deploy proccess.

Reply:
Ahh, right